var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -434,6 +435,8 @@ def configure_args(self):
             else:
                 config_args.append('--without-cuda')
 
+        if '+cxx_exceptions' in spec:
+                config_args.append('--enable-cxx-exceptions')
```

Comment:
Can you explicitly disable this in an else-clause? You never know when the defaults change. Also, this is indented too far.